cypress/support/constants.ts

```diff
@@ -2,10 +2,244 @@ const GOOD_PASSWORD = "password123#";
 const BAD_PASSWORD_LENGTH = "password12#";
 const BAD_PASSWORD_NO_NUMBER = "password####";
 const BAD_PASSWORD_NO_SYMBOL = "password1234";
+const CONFIG_INTEGRATIONS_AUTOMATIONS = {
```

Comment:
reused on 
- settings.spec.ts to test create/edit/delete integration
- policiesflow.spec.ts to test create/edit/remove fail policy automation
- software.spec.ts to test create/edit/remove software automation
- premium/admin.spec.ts to test global admin access to set policy automation for a team